src/Workleap.Extensions.MediatR.Analyzers.Tests/Workleap.Extensions.MediatR.Analyzers.Tests.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net462;net6.0</TargetFrameworks>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Since MediatR upgraded the BCL async interfaces to 8.0.0, there was an MSBuild issue when running the analyzers tests:

![image](https://github.com/gsoft-inc/wl-extensions-mediatr/assets/14242083/1b7a38af-f326-47e2-bad0-18205dd40d65)

I gave up after 2 hours and finally removed the tests for .NET 4.6.2. I'm quite confident about the fact that the analyzers still works with .NET 4.6.2 tho.